contracts/v6/OraclePrice.sol

```diff
@@ -1,58 +1,57 @@
 // SPDX-License-Identifier: Unlicensed
 pragma solidity ^0.6.12;
+pragma experimental ABIEncoderV2;
 
 import "./interface/KeeperCompatibleInterface.sol";
-import "./interface/OmsPolicy.sol";
+import "./interface/IOmsPolicy.sol";
 import "./interface/EACAggregatorProxy.sol";
 import "./library/SafeMath.sol";
 import "./common/Ownable.sol";
 
 contract OraclePrice is Ownable, KeeperCompatibleInterface {
     using SafeMath for uint256;
+
+    struct OracleInfo {
```

Comment:
Looks like you've defined this in OraclePriceStruct. I assume it best to just use that one?

Reply:
Actually OraclePriceStruct is in solidity v4 and our oracle price contract is in v6 that's why i used oracleInfo struct inside code.

Reply:
As the calculateReferenceRate is in the OraclePrice now, this is no longer an issue.